zabbix/umbrel-app.yml

```diff
@@ -0,0 +1,36 @@
+manifestVersion: 1.1
+id: zabbix
+category: networking
+name: Zabbix
+tagline: The all-in-one, open-source solution that lets you monitor anything
+icon: https://raw.githubusercontent.com/deralia/umbrel-app-store/master/deralia-zabbix/assets/icon.png
+version: "7.0.3"
+description: >-
+  A Zabbix Server + Frontend interface to monitor your network.
+
+
+  The first time running Zabbix might take a few minutes because it needs to initialize the database. Please be patient.
+
+
+  The Zabbix Agent listens on port 10050, the Zabbix Server listens on port 10051 and Zabbix Frontend listens on port 10052.
```

Comment:
Good catch on the two newlines 😀

Reply:
Thanks, I guess :-)

I noticed this “requirement” while testing it for my community repo

Resolving this conversation and waiting for more feedback on the other one